.azure-pipelines/client.yml

```diff
@@ -66,7 +67,18 @@ jobs:
           version: '$(DotNetCoreVersion)'
 
       - task: DotNetCoreCLI@2
-        displayName: 'Build & Test'
+        displayName: 'Build & Test (no live tests)'
+        condition: eq(variables['System.TeamProject'], 'public')
         inputs:
           command: test
           projects: '$(ProjectFile)'
+          arguments: --filter TestCategory!=Live
+        
+      - task: DotNetCoreCLI@2
+        displayName: 'Build & Test (with live tests)'
```

Comment:
Just for clarification this runs all tests correct? Or does it only run live tests? For example does it also run the recorded tests?